### PR TITLE
Test folder refactor

### PR DESCRIPTION
**Pull request by paulirish**
This is a followup to the large folder refactor PR (#243).

-----------------

* Pretty straightforward path adjustments. 
* Moved our fixture `manifest.json` to hang out in `fixtures/` now.
* Added a little `watch` script that I use to quickly get told about failures. Dependent on [`entr`](http://entrproject.org/): `brew install entr`
* Documented said `watch` script. Some whitespace edits snuck into the diff.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "closure": "closure/closure-type-checking.js",
     "lint": "eslint .",
     "test": "npm run lint --silent && npm run unit && npm run closure",
+    "watch": "find . -name '*.js' | grep -v 'node_modules' | grep -v 'extension' | entr npm run unit",
```
Comment (samccone) [inline package.json]: not everyone has entr... :)

Comment (samccone) [inline package.json]: but it is doc'd so that is cool then

File: test/src/audits/html/theme-colors.js [hunk 2]
```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const Audit = require('../../../src/audits/html/meta-theme-color.js');
+const Audit = require('../../../../src/audits/html/meta-theme-color.js');
```
Comment (brendankenny) [inline test/src/audits/html/theme-colors.js]: needs more `../../../../../../../../`

**Comment by brendankenny** [comment]
☑️🔀LGTM

**Pull request merged**